packages/core/src/engine/Autodiff.test.ts

```diff
@@ -370,7 +370,7 @@ describe("polyRoots tests", () => {
     expect(gradient[2]).toBeCloseTo(-1 / 3);
   });
 
-  test("quintic", () => {
+  test("quartic", () => {
```

Comment:
Hey @jiriminarcik, I just noticed that you erroneously changed this test description in this PR. The polynomial is indeed quintic, since the fifth-degree coefficient is implicitly 1. I'm opening a PR to change this line back.